ffi/diplomat/ffi_coverage/src/main.rs

```diff
@@ -241,6 +241,10 @@ lazy_static::lazy_static! {
         // could be exposed in the future but it's complicated
         "icu::plurals::rules",
 
+        // Pulls in libstd, which we'd rather not do, and it's not a super prominent
+        // API
```

Comment:
Nit: Don't editorialize on the "prominence" of the APIs

```suggestion
        // Pulls in libstd, which we'd rather not do
```